src/platform/site-wide/cta-widget/index.js

```diff
@@ -118,6 +117,23 @@ export class CallToActionWidget extends React.Component {
   };
 
   getHealthToolContent = () => {
+    if (this.isAccessible()) {
```

Comment:
Swapped this with the inaccessible state handling, since I'm concerned about the possibility of two async requests to upgrade will result in one of them having an error despite having succeeded once and writing all the updated data. As long as the user's account details passes all the requirements, we should allow the user to go to the tool.